projects/storefrontstyles/scss/themes/calydon/layout/page-templates/_category-page.scss

```diff
@@ -0,0 +1,13 @@
+%CategoryPageTemplate-calydon {
```

Comment:
This must be a generic behaviour of the banner component, rather than a page template specific solution. Can we move this to banner.

Reply:
A ticket for this will be created.

Reply:
@tobi-or-not-tobi @kacperknapik Agreed and ticket #6711 created, it will be addressed during the `calydon-as-base` effort to avoid delaying the merge of this PR any further.